assignment.md

```diff
@@ -52,6 +52,9 @@
   - votes should identified by IP address and unique
 - add the option to make commenting and voting realtime
   - via GraphQL Subscriptions or WebSockets
+- present your ability to test the code
+  - you don't have to test everything
+  - just add few unit, integration and E2E tests
```

Comment:
Yes! Thanks 😅

E2E are nice to have but integration test are pretty common on projects 🤷🏻‍♂️ but let's keep them optional. If we find out that candidates do just the minimum required stuff and we need to know that they can do integration testing we can just change it.

Reply:
Ok, cool! I'm gonna try pressing the Merge pull request button to find if I have sufficient rights then, because I don't think anyone else is taking a look at this 😄